chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/Edition2014ListingValidator.java

```diff
@@ -147,6 +152,7 @@
             comparisonReviewers.add(chplNumberComparisonReviewer);
             comparisonReviewers.add(devBanComparisonReviewer);
             comparisonReviewers.add(listingStatusAndUserRoleReviewer);
+            comparisonReviewers.add(realWorldTestingReviewer);
```

Comment:
RWT isn't valid for 2014 Edition listings, right? Why review them here? Or, if they need to be reviewed, shouldn't it be a different reviewer? One that rejects anything with that data?

Reply:
We need to make sure that the user has not passed in any RWT information.  Based on the reviewer checking for a rwtEligibility year, which can only exist for a 2015 listing,by adding this reviewer to 2014, it will always reject the data if it is passed in.

Reply:
I think the RWT reviewer should check against the Listing's edition. We can't know, for sure, that only 2015 Edition Listings get that rwt eligibility date. I know the code that adds the date as a value filters to 2015, but with defensive coding practices we ought to check from this end as well